tools/coverage/calc/base.go

```diff
@@ -48,26 +48,6 @@ func (blk *codeBlock) addToFileCov(coverage *Coverage) {
 	}
 }
 
-// add blk Coverage to file group Coverage; return true if the row is concerned
-func updateConcernedFiles(concernedFiles *map[string]bool, filePath string,
-	isPresubmit bool) (isConcerned bool) {
-	// get linguist generated attribute value for the file.
-	// If true => needs to be skipped for coverage.
-	isConcerned, exists := (*concernedFiles)[filePath]
-
-	if !exists {
-		if isPresubmit {
-			// presubmit already have concerned files defined,
-			// we don't need to check git attributes here
-			isConcerned = false
-			return
-		}
-		isConcerned = !git.IsCoverageSkipped(filePath)
-		(*concernedFiles)[filePath] = isConcerned
-	}
-	return
-}
-
 // add blk Coverage to file group Coverage; return true if the row is concerned
```

Comment:
Funny, this is exactly the same comment used in the function you moved. @steuhs what are the right comment for these functions?

Reply:
updated